markup/asciidocext/asciidocext_config/config.go

```diff
@@ -64,6 +71,13 @@ var (
 		"manpage":   true,
 	}
 
+	AllowedDocType = map[string]bool{
```

Comment:
I did not see a usecase for these doctypes in Hugo context. If there is a usecase, we should mention them in the documentation.

Reply:
I don't have a use case for it either, but it was already in the type `asciidocext_config.Config` without being used at all, so I just added the missing parts. I'll remove it completely in that case.